build.gradle

```diff
@@ -189,6 +189,9 @@ subprojects {
         /* Ballerina OpenAPI */
         ballerinaOpenAPILibs "io.ballerina:module-ballerina-openapi:${openapiVersion}"
 
+        /* Ballerina AsyncAPI */
+        ballerinaStdLibs "io.ballerina:module-ballerina-asyncapi:${asyncapiVersion}"
```

Comment:
Shall we use `ballerinaC2cLibs` config? We will rename that as `ballerinaExtensions`

Reply:
Sure, Thanks for the suggestion. Resolved